pyproject.toml

```diff
@@ -0,0 +1,98 @@
+[tool.poetry]
+name = "saleor"
+version = "2.4.0"
+description = "A modular, high performance e-commerce storefront built with GraphQL, Django, and ReactJS."
+authors = ["Mirumee Software <hello@mirumee.com>"]
+license = "BSD-3-Clause"
+
+readme = "README.md"
+
+homepage = "https://getsaleor.com/"
+repository = "https://github.com/mirumee/saleor"
+documentation = "https://docs.getsaleor.com/en/latest/"
+
+[tool.poetry.dependencies]
+python = ">=3.5, <3.8"
+Babel = "^2.6"
+bleach = "<3"
+boto3 = "^1.9"
+django = "~2.1"
+dj-email-url = "^0.1.0"
+dj-database-url = "^0.5.0"
+django-babel = "^0.6.2"
+django-bootstrap4 = "^0.0.7"
+django-cache-url = "^3.0"
+django-countries = "^5.3"
+django-filter = "^2.1"
+django-fsm = "^2.6"
+django-graphql-jwt = "^0.2.1"
+django-impersonate = "^1.4"
+django-measurement = "~3.0"
+django-mptt = "^0.9.1"
+django-phonenumber-field = "^2.2"
+django-prices = "^1.0"
+django-prices-openexchangerates = "^1.0"
+django-prices-vatlayer = "^1.0"
+django-redis = "^4.10"
+django-storages = "^1.7"
+django-templated-email = "^2.3"
+django-versatileimagefield = "^1.10"
+django-webpack-loader = "^0.6.0"
+django-celery-results = "^1.0"
+django-elasticsearch-dsl = "~0.5"
+django-recaptcha = "^1.0"
+django-silk = "^2.0"
+elasticsearch = "~6.3"
+elasticsearch-dsl = "~6.0"
+faker = "^1.0"
+google-measurement-protocol = "^1.0"
+google-i18n-address = "^2.3"
+graphene-django = "^2.2"
+Markdown = "^3.0"
+maxminddb = "^1.4"
+maxminddb-geolite2 = "^2018.703"
+phonenumberslite = "^8.10"
+prices = "^1.0"
+psycopg2-binary = "^2.7"
+purl = "^1.4"
+raven = "~6.9"
+requests = "^2.21"
+social-auth-app-django = "^3.1"
+text-unidecode = "^1.2"
+uwsgi = {version = "^2.0", platform = "!= win32, != cygwin"}  # exclude windows
+celery = {version = "^4.1", extras = ["redis"]}
+redis = "<3.0.0"
+kombu = "<4.3.0"   # kombu version is pinned to avoid issues described here https://github.com/celery/celery/issues/5369
+weasyprint = "^45.0"
+graphene-django-optimizer = "^0.3.6"
+braintree = "~3.49"
+razorpay = "^1.1"
+stripe = "^2.21"
+urllib3 = "~1.23"
```

Comment:
Do we really need this?

Reply:
Not too sure. It was in the Pipfile, so I kept it as it was.

Reply:
I don't have it in my own fork, and everything is fine.